app/code/Magento/Checkout/view/frontend/web/js/view/billing-address.js

```diff
@@ -92,6 +92,7 @@ function (
                 } else {
                     this.isAddressSameAsShipping(
                         newAddress != null &&
+                        window.isbothAddressSame &&
```

Comment:
What the reason to provide global variable? You can use local class context (this) for it.